### PR TITLE
Add activation class

### PR DESCRIPTION
**Pull request by ramiabraham**
Adds the affwp activation class, adds an includes dir, moves admin settings file to includes/admin. Implements #14.

### CODE REVIEW

File: includes/class-activation.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+     * Display notice if AffiliateWP isn't installed
+     *
+     * @access      public
+     * @since       1.0.0
+     * @return      string The notice to display
+     */
+    public function missing_affiliatewp_notice() {
+
+        if ( $this->has_affiliatewp ) {
+           echo '<div class="error"><p>' .  $this->plugin_name . sprintf( __( ' requires %sAffiliateWP%s. Please activate it to continue.', 'affiliatewp-order-details-for-affiliates' ), '<a href="http://affiliatewp.com/" title="AffiliateWP" target="_blank">', '</a>' ) . '</p></div>';
```
Comment (michaelbeil) [inline includes/class-activation.php]: i think you meant to change affiliatewp-order-details-for-affiliates to affiliatewp-store-credit

File: includes/class-activation.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+        } else {
+            echo '<div class="error"><p>' . $this->plugin_name . sprintf( __( ' requires %sAffiliateWP%s. Please install it to continue.', 'affiliatewp-order-details-for-affiliates' ), '<a href="http://affiliatewp.com/" title="AffiliateWP" target="_blank">', '</a>' ) . '</p></div>';
```
Comment (michaelbeil) [inline includes/class-activation.php]: here too

Comment (ramiabraham) [inline includes/class-activation.php]: yikes, good catch!

File: includes/class-activation.php (hunk 1 above)
Comment (ramiabraham) [inline includes/class-activation.php]: ✅

**Pull request merged**